cmd/server/main.go

```diff
@@ -519,6 +527,11 @@ func (p *Proxy) runAgentServer(o *ProxyRunOptions, server *server.ProxyServer) e
 func (p *Proxy) runAdminServer(o *ProxyRunOptions, server *server.ProxyServer) error {
 	muxHandler := http.NewServeMux()
 	muxHandler.Handle("/metrics", promhttp.Handler())
+	muxHandler.HandleFunc("/debug/pprof", pprof.Index)
```

Comment:
This is enabling a lot more than just contention profiling indicated by the flag.

```
Count | Profile
-- | --
0 | allocs
0 | block
0 | cmdline
12 | goroutine
0 | heap
0 | mutex
0 | profile
13 | threadcreate
0 | trace
```

Reply:
Good point. Added --enable-profiling.